NEWS.md

```diff
@@ -86,6 +90,8 @@ plumber 1.0.0
 
 ### Bug fixes
 
+* Fix OpenAPI server url error when `HTTP_REFERER` is NULL. (@meztez, #562)
```

Comment:
This doesn't seem worth mentioning (has anyone filed an issue about it?)

```suggestion
```

Reply:
Removing. #421.